lib/internal/Magento/Framework/Backup/AbstractBackup.php

```diff
@@ -296,4 +306,21 @@ protected function _filterName($name)
 
         return $name;
     }
+
+    /**
+     * Set if keep files of backup
+     *
+     * @param boolean $keepSourceFile
+     * @return $this
+     */
+    public function setKeepSourceFile($keepSourceFile)
```

Comment:
Let's add a new interface for these methods.

Reply:
Added to `SourceFileInterface`